packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/Point.java

```diff
@@ -4,12 +4,14 @@
 
 package io.flutter.plugins.camera.features;
 
+import androidx.annotation.NonNull;
+
 /** Represents a point on an x/y axis. */
 public class Point {
   public final Double x;
   public final Double y;
 
-  public Point(Double x, Double y) {
+  public Point(@NonNull Double x, @NonNull Double y) {
```

Comment:
Talked offline and realized that these values do seem nullable. There is actually a test to handle the scenario: https://github.com/flutter/packages/blob/main/packages/camera/camera_android/android/src/test/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeatureTest.java#L84.

Looking throughout the api though, it doesn't look like the value could actually be pass as null.

Public facing feature usages:
https://github.com/flutter/packages/blob/main/packages/camera/camera/lib/src/camera_controller.dart#L677
https://github.com/flutter/packages/blob/main/packages/camera/camera/lib/src/camera_controller.dart#L814

In a separate PR, this may be worth changing to `double`s. I don't want to accidentally break something in this PR that is just fixing lints.